packages/std/src/query.rs

```diff
@@ -150,4 +150,25 @@ mod staking {
         pub accumulated_rewards: Coin,
         // TODO: do we want to expose more info?
     }
+
+    /// Billionth represents a fixed-point decimal value with 9 fractional digits.
```

Comment:
The description makes sense to me, the name is hard to understand I think. E.g. `Billionth::one()` reads as "one billionths".

Naming siggestion: `Decimal9` – "a fixed-point decimal value with 9 fractional digits.". Then `Decimal9::one()` is easier to interprete as `1.0 as Decimal9`

Reply:
Sure, I will rename